src/components/Pages/Login/LoginForm/index.js

```diff
@@ -43,22 +45,47 @@ export const LoginForm = () => {
     e.preventDefault();
     //Validate user inputs
     if (email === "" || password === "") {
-      return setError("Please fill both fields");
+      return;
+      // setError("Please fill both fields");
     }
 
-    if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(email) === false) {
-      return setError("Invalid email address");
+    if (
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
```

Comment:
Store the regex patterns inside variables. It's easier for code readability. 
i.e.
`const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/`

`emailRegex.test(email)`